array/DNAm/preprocessing/QCwithinCellType.rmd

```diff
@@ -18,21 +18,42 @@ library(pheatmap)
 library(pander)
 library(RColorBrewer)
 library(kableExtra)
+library(data.table)
 
 
-source(args[3]) ### change the content of this file to run QC with different thresholds
-
 dataDir <- args[2]
 setwd(dataDir) 
 
+refDir <-args[3]
 
 gdsFile <-paste0(dataDir, "/2_gds/raw.gds")
+configFile <- paste0(dataDir, "/config.r")
+epic2Manifest <- paste0(refDir,"/EPICArray/EPIC-8v2-0_A1.csv")

```

Comment:
do we need corresponding code here for load manifests for other files? Also do we need to state somewhere what these files are. Do we need to include them in the repo.